mbtools/models.py

```diff
@@ -120,39 +141,74 @@ def delete_empty_categories(self):
                 elem.getparent().remove(elem)
 
 
+class MoodleLessonPage:
+    def __init__(self, tree, location):
+        self.tree = tree
+        self.location = location + f' (page: {self.name()})'
+        self.id = int(self.tree.attrib['id'])
+
+    def name(self):
+        return self.tree.xpath("title")[0].text
+
+    def next(self):
+        return int(self.tree.xpath("nextpageid")[0].text)
+
+    def prev(self):
+        return int(self.tree.xpath("prevpageid")[0].text)
+
+    def answers(self):
+        answer_objs = []
+        for answer in self.tree.xpath("answers/answer"):
+            answer_objs.append(
+                MoodleLessonAnswer(answer, self.location)
+            )
+        return answer_objs
+
+    def html_element(self):
+        return MoodleHtmlElement(self.tree.xpath("contents")[0], self.location)
+
+    def html_elements(self):
+        elems = []
+        elems.append(self.html_element())
+        for answer in self.answers():
+            elems.append(answer.html_element())
+        return elems
+
+
+class MoodleLessonAnswer:
+    def __init__(self, tree, location):
+        self.tree = tree
+        self.location = location + f' (answer id: {self.tree.attrib["id"]})'
+
```

Comment:
I agree with you here. In verification I think we were trying to find a way to differentiate between different sections of html but this id value doesn't even show up in the UI

Reply:
@mjoss17 - It occurred to me that while the ID value itself isn't particularly useful, it is good to know that the issue is in the answer HTML and not the page content itself so at least the user knows to look there (otherwise it may be confusing). Given that my suggestion above may have been misguided. What do you think about making this location something like:

```python
 @property
 def location(self):
        return f"{self.lesson_page.location}: Answer value"
```

I also realized we are currently not parsing for answer responses (I'll be creating a new issue around that), but when we add those it's location string could then be something like `f"{self.lesson_page.location}: Answer response"`